frontend/libs/studio-components/src/components/StudioTable/StudioTable.tsx

```diff
@@ -0,0 +1,12 @@
+import type { ComponentProps } from 'react';
+import React, { forwardRef } from 'react';
+import { Table } from '@digdir/designsystemet-react';
+import type { WithoutAsChild } from '../../types/WithoutAsChild';
+
+export type StudioTableProps = WithoutAsChild<ComponentProps<typeof Table>>;
+
+export const StudioTable = forwardRef<HTMLTableElement, StudioTableProps>((props, ref) => (
+  <Table data-size='sm' {...props} ref={ref} />
```

Comment:
I suggest removing the `data-size` here since it is already set to `sm` on the body for the entire solution (as William mentioned in my PR here: https://github.com/Altinn/altinn-studio/pull/15255#discussion_r2063209811)

```suggestion
  <Table {...props} ref={ref} />
```

Reply:
true, thx!